desk/app/groups.hoon

```diff
@@ -2811,7 +2811,18 @@
     ::  consider it uninitialized.
     ::
     =.  net  [%sub *@da |]
-    (go-start-updates ?~(why | &))
+    (go-start-updates ?~(error | &))
+  ::  +go-lost-admin: adjust the group state when admin rights were revoked
+  ::
+  ++  go-lost-admin
+    ^+  go-core
+    =.  group
+      %_  group
+        tokens.admissions    ~
+        pending.admissions   ~
+        requests.admissions  ~
+      ==
+    go-core
```

Comment:
This could just be a `%_  go-core` with the same edits I think, no need to `=.` in the context and produce it separately. Alternatively, this could be smaller if you `=.  admissions.group`, maybe.